src/GenericPlanForm.tsx

```diff
@@ -141,15 +152,20 @@ export class GenericPlanForm extends React.Component<IProps, IState> {
                           </Grid>
                           <Grid item justify="space-evenly" spacing={1} xs={6}>
                             <List dense={true}>
-                              {this._get_widget_list(parameterObject)}
+                              {this.getWidgetList(parameterObject)}
                             </List>
                           </Grid>
-                          <Grid item justify="flex-end" xs={3}>
-                            {parameterObject.name.slice(-1) === 's' ?  <IconButton onClick={() => this._addParameter(parameterObject.name)}>
-                                                          <AddCircleOutlineIcon />
-                                                        </IconButton>
-                                                        :<IconButton/>}
-                          </Grid>
+                            {parameterObject.name.slice(-1) === 's' ?
```

Comment:
I forgot to ask in a previous PR when I saw it, but what are we looking for, and what is the full `parameterObject.name`?

Reply:
Parameters have an attribute called `isList`, but currently the server isn't giving us that information.  So a temporary fix for now is to assume the parameter is a list if the name ends in 's'.

Reply:
OK, I see. Thanks for the explanation.